config/context/application.properties

```diff
@@ -0,0 +1,5 @@
+credential.accountId=test@notary
+credential.pubkey=092e71b031a51adae924f7cd944f0371ae8b8502469e32693885334dedcc6001
+credential.privkey=e51123b78d658418d018e7d2486021209af3cff82714b4cb7925870fec6097dc
```

Comment:
This doesn't look very secure

Reply:
Agree but I haven't got any other decisions so far. Would appreciate if you have something to say. Note the contents are just testing purpose keys.